src/app/pages/about-pycon/about-pycon.page.html

```diff
@@ -31,7 +31,10 @@
       <br>
       <ion-col>
         <ion-text>App Version: {{liveUpdateService.build}}</ion-text>
+        <br>
         <ion-button *ngIf="loggedIn" (click)="shareDebug()">Share Debug Info with PyCon Staff</ion-button>
+        <br>
+        <ion-text>Environment: {{environmentUrl}}</ion-text>
```

Comment:
```suggestion
        <ion-text *ngIf="environmentUrl !== 'https://us.pycon.org'">Environment: {{environmentUrl}}</ion-text>
```

Maybe? I'm on the fence.

Reply:
Seems fine either way to me. I don't think we need much secret :-p